src/Core/Managed/Shared/Utils.cs

```diff
@@ -74,6 +74,20 @@ public static TimeSpan ValidateDuration(string value)
             return interval;
         }
 
+        /// <summary>
+        /// Returns min DateTimeOffset value if not a valid DateTimeOffset.
+        /// </summary>
+        public static DateTimeOffset ValidateDateTimeOffset(string value)
+        {
+            DateTimeOffset timestamp;
+            if (!DateTimeOffset.TryParse(value, CultureInfo.InvariantCulture, DateTimeStyles.AssumeLocal, out timestamp))
```

Comment:
This is used for validating TestTimeStamp and we don't know if the caller is passing us a UTC offset or not.  There is precedent in the SDK set since RequestTelemetry is already using AssumeLocal for their start/end-time and timestamp.  Is there a better way to enforce passing a timezone denoted timespan when its assigned?